### DISCUSSION TITLE
[预览] v2.8.4

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4414] [分享 / Share]
包含 [v2.7.6](https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.6) 的所有更新内容.

- `直播间网页全屏自适应` 支持自定义侧边栏宽度. (#4216)
- 修复 `直播勋章快速更换` 获取头衔列表时报错. (#4403)

由于更换了底层动态 API, 本次更新需要将**脚本本体**和**动态相关组件** (自定义顶栏 / 清爽首页 / 极简首页) 均更新至最新, 才能正常运行, 不要再来问为什么动态空白 / 报错了.

最新版本号参考:
|                        | v2.7.6 (GitHub 源)  | v2.7.6 (jsDelivr 源) | 预览版 v2.8.4 (GitHub 源)     |
| ---------------------- | ------------------- | -------------------- | ----------------------------- |
| 本体版本               | v2.7.6-9-g881ccf2f0 | v2.7.6-59-g3d43a4e0c | v2.8.4-preview-103-g6049a6149 |
| 组件版本 (Commit Hash) | `881ccf2f0`         | `3d43a4e0c`          | `6049a6149`                   |

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.4-preview'>[预览] v2.8.4</a>.</em>

**Comment by kethoven** [comment]
bug反馈。
自定义顶栏-动态，点击稍后再看，图标无变化。即使已经添加到稍后再看列表，图标也不改变。
